src/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ ExternalProject_Add(UFS_UTILS
   )
 
 if(NOT CCPP_SUITES)
-  set(CCPP_SUITES "FV3_CPT_v0,FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_regional,FV3_GSD_SAR,FV3_GSD_v0,FV3_GFS_v15p2,FV3_GFS_v16,FV3_RRFS_v1beta,FV3_HRRR,FV3_RRFS_v1alpha,FV3_GFS_v15_thompson_mynn_lam3km")
+  set(CCPP_SUITES "FV3_CPT_v0,FV3_GFS_2017_gfdlmp,FV3_GFS_2017_gfdlmp_regional,FV3_GFS_v15p2,FV3_GFS_v16,FV3_RRFS_v1beta,FV3_HRRR,FV3_RRFS_v1alpha,FV3_GFS_v15_thompson_mynn_lam3km")
```

Comment:
Can we do the removal of this suite in a separate PR (just like we did in the corresponding `regional_workflow` SPP PR#[685](https://github.com/ufs-community/regional_workflow/pull/685))?

Reply:
@gsketefian, if we leave these suites in the CMakeLists.txt file, the build will fail, since they are no longer available in ufs-weather-model hash that we're updating to.  Leaving them in the regional_workflow code is OK for now, but they won't be available for use anymore, due to the hash update of ufs-weather-model.

Reply:
Ok then, no problem